engine/src/cubos/engine/renderer/deferred_renderer.cpp

```diff
@@ -747,6 +776,14 @@ void DeferredRenderer::onRender(const Camera& camera, const RendererFrame& frame
         mSsaoTexBp->bind(mSsaoTex);
         mSsaoTexBp->bind(mSampler);
     }
+    mUVScaleBp->setConstant(
+        glm::vec2((float)camera.viewportSize.x / (float)mSize.x, (float)camera.viewportSize.y / (float)mSize.y));
+    mUVOffsetBp->setConstant(glm::vec2((float)camera.viewportPosition.x / (float)mSize.x,
+                                       (float)camera.viewportPosition.y / (float)mSize.y));
+    mSkyGradientBottomBp->setConstant(frame.skyGradient(0));
+    mSkyGradientTopBp->setConstant(frame.skyGradient(1));
+    mInvVBp->setConstant(glm::inverse(mvp.v));
+    mInvPBp->setConstant(glm::inverse(mvp.p));
```

Comment:
This function explains each step with a comment, here should not be an exception

Reply:
It's part of the same step as the calls above - setting the pipeline state.